src/VisualStudio/CSharp/Impl/Options/AutomationObject.cs

```diff
@@ -318,14 +324,14 @@ public int RenameTrackingPreview
 
         public int ShowKeywords
         {
-            get { return GetBooleanOption(CompletionOptions.IncludeKeywords); }
-            set { SetBooleanOption(CompletionOptions.IncludeKeywords, value); }
+            get { return 0; }
```

Comment:
We can look at telemetry... but i'm not seeing how it would reveal much of anything.  For example, someone might have it turned off simply because they wanted to turn off all completion.  Not because they were saying "when i bring up completion on ctrl-space then i don't want keywords in it".

Note that not having keywords in the list is also quite detrimental if you do want completion.  Say you do hte following:

```void Foo($$```

No keywords means that if you bring up completion you won't have "int, string, etc." in that list.   In which case, the list is an active hindrence.

It is my opinion that this option literally serves no value.  It clearly has to be on when completion-on-typing is on.  And if that option is off, then there is still no actual user value provided by keeping keywords out of hte list.

What is the customer argument for why they'd want this?  The list of keywords is already accurate and appropriate in each language.  It does not clutter the focused lists we bring up after things like ```<dot>```.  It's only a historical oddity that we even bothered with this option in the first place.  And that history was also because we were a lot more aggressive and inaccurate with keywords when we original introduced them.  

IMO, this is an example of an option for an options sake.  Unlike some options, where there is actual a use-case (though the audience is small), this option has no use-case.